### PR TITLE
Build self-contained Convert-BpfToNative.ps1

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Signed-off-by: Alan Jowett <alan.jowett@microsoft.com>

## Description

1) Make Convert-BpfToNative.ps1 fully self-contained (except for bpf2c.exe).
2) Update nuget package to include ebpfapi.dll (bpf2c.exe depends on it).

## Testing

CI/CD

## Documentation

No.

### CODE REVIEW

File: tests/bpf2c_tests/elf_bpf.cpp [hunk 1]
```diff
@@ -122,7 +122,7 @@ run_test_elf(const std::string& elf_file, _test_mode test_mode)
 
             REQUIRE(raw_result.size() == raw_output.size());
             for (size_t i = 0; i < raw_result.size(); i++) {
-                REQUIRE(raw_output[i] == raw_result[i]);
+                REQUIRE(((i >= 0) && (raw_output[i] == raw_result[i])));
```
Comment (dthaler) [inline tests/bpf2c_tests/elf_bpf.cpp:125]: Why is this change needed?  size_t is unsigned so i>=0 should always be true, no?

**Review by dthaler — DISMISSED** [review]

File: tests/bpf2c_tests/elf_bpf.cpp (hunk 1 above)
Comment (Alan-Jowett) [reply tests/bpf2c_tests/elf_bpf.cpp]: My bad. This was added to include the line number in the debug message when the test failed. Removed as it's no longer needed.

Comment (Alan-Jowett) [reply tests/bpf2c_tests/elf_bpf.cpp]: My bad. This was added to include the line number in the debug message when the test failed. Removed as it's no longer needed.

**Review by dthaler — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Comment by shankarseal** [comment]
IMHO embedding vcxproj into PS1 that goes into nuget package, is complex. Couldnt we have a simpler solution such as downloading the template vcxproj from github repo (with expected version) or putting the vcxproj as a DLL resource?

**Review by saxena-anurag — APPROVED** [review]
